bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBinding.java

```diff
@@ -108,7 +108,7 @@ public void removeKNXTypeMapper(KNXTypeMapper typeMapper) {
     @Override
     protected void internalReceiveCommand(String itemName, Command command) {
         logger.trace("Received command (item='{}', command='{}')", itemName, command.toString());
-        if (!isEcho(itemName, command)) {
+        if (!KNXConnection.getCompatibilityOH2() && !isEcho(itemName, command)) {
```

Comment:
> One thing I can say, the autoupdate function in KNX 1.8 binding is never ever called, running in OH1 nor OH2.
> Why? Is this behavior expected?

I just checked this. The reason is that it isn't registered as an auto update provider. With [this change](https://github.com/openhab/openhab/pull/4728), it actually IS called. But does this mean that this code has never been active even in OH1? If so, I do not know, what effect https://github.com/openhab/openhab/pull/4728 might have on OH1 installations...